packages/twenty-front/src/modules/object-record/record-table/record-table-cell/components/RecordTableCellEditButton.tsx

```diff
@@ -19,10 +20,10 @@ export const RecordTableCellEditButton = ({
   Icon,
 }: RecordTableCellEditButtonProps) => (
   <StyledEditButtonContainer
```

Comment:
we could also justify extracting the whole StyledEditButtonContainer into its own file instead of importing AnimationDivProps